aries_vcx/src/handlers/proof_presentation/verifier/states/presentation_request_set.rs

```diff
@@ -1,12 +1,22 @@
-use crate::messages::proof_presentation::presentation_request::PresentationRequestData;
+use crate::handlers::proof_presentation::verifier::states::presentation_request_sent::PresentationRequestSentState;
+use crate::messages::proof_presentation::presentation_request::PresentationRequest;
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
-pub struct PresentationRequestSet {
-    pub presentation_request_data: PresentationRequestData,
+pub struct PresentationRequestSetState {
```

Comment:
The other state names are postfixed with `State`, so I applied to convention here too.